js/ui/payg.js

```diff
@@ -510,5 +510,11 @@ var PaygNotifier = new Lang.Class({
         this._notification.connect('destroy', () => {
             this._notification = null;
         });
+
+    },
+
+    clearNotification() {
```

Comment:
This code is copied from near the top of the previous function: could you make the previous function call this new function instead?

Reply:
Ah, yes! will do :)